src/Docker/Docker.php

```diff
@@ -80,7 +73,7 @@ public function getImageManager()
 
    /**
      * Show the docker components version information
-     * @return json object with version values
```

Comment:
Not a valid type (see http://www.phpdoc.org/docs/latest/references/phpdoc/types.html)